### PR TITLE
Add dataprep microservice to chatQnA example

### PR DESCRIPTION
**Pull request by zhlsunshine**
## Description

Change list:

1. Add GMC config samp yaml files both on xeon and gaudi platform
2. Add e2e test data on repo: gaudi.txt which includes the key features of Gaudi description.

## Issues

https://github.com/opea-project/GenAIInfra/issues/168
https://github.com/opea-project/GenAIInfra/issues/207

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

 `n/a`.

## Tests

This is for the e2e test.

### CODE REVIEW

File: microservices-connector/test/data/gaudi.txt
Comment (KfreeZ) [inline microservices-connector/test/data/gaudi.txt:1]: there is supposed to have a script/doc to show how to upload this "gaudi.txt", I assume it will be another PR for it, right?

**Review by KfreeZ — APPROVED** [review]

File: microservices-connector/test/data/gaudi.txt
Comment (zhlsunshine) [reply microservices-connector/test/data/gaudi.txt:1]: Hi @KfreeZ, right, I have to upload `gaudi.txt` firstly, then the e2e script can get it.

**Review by daisy-ycguo — COMMENTED** [review]
No e2e tests?

**Comment by zhlsunshine** [comment]
> No e2e tests?

Hi @daisy-ycguo, yeah, I have to upload `gaudi.txt` to the repo firstly, then the e2e test script can get it from the repo via online link.

**Review by daisy-ycguo — APPROVED** [review]
lgtm
